composer.json

```diff
@@ -15,7 +15,8 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0",
+        "php": ">=5.4",
+        "ext-intl": "*",
```

Comment:
I would move intl extension to require-dev section because it is required only in tests until someone uses the new keymaker in own code